rust/geoarrow-schema/src/crs.rs

```diff
@@ -67,6 +67,14 @@ impl Crs {
         }
     }
 
+    /// Construct from an opaque string identifier
+    pub fn from_srid(value: String) -> Self {
```

Comment:
```suggestion
    pub fn from_srid(value: impl Into<String>) -> Self {
```

This means that we can pass in integers here as well, because they'll be automatically converted to string

Reply:
Oh hmm `impl From<usize> for String` isn't actually implemented; just `impl Display for usize`, and we don't want to allow anything that can be displayed, so I guess we'll hace to keep this as it is.